src/sub.c

```diff
@@ -1124,6 +1124,35 @@ natsSubscription_QueuedMsgs(natsSubscription *sub, uint64_t *queuedMsgs)
     return s;
 }
 
+natsStatus
+natsSubscription_GetInfo(natsSubscription* sub, int64_t* sid, char* subject)
```

Comment:
Although I like reducing the number of APIs, what if some day we think that we should return some more information from the subscription? If we think that may happen, then we should here have a GetSubject() and GetID() right out of the get go.

Reply:
I was also debating whether to split this into 2 separate API's or not. And it is indeed a bit random that these 2 unrelated fields are returned together.

I split them as you suggested.